src/routes/productRoutes.ts

```diff
@@ -36,6 +36,12 @@ productRouter.get(
 	productController.read_single_product,
 );
 
+productRouter.get(
+	"/:id/recommended",
+	userAuthentication.authenticateUser,
```

Comment:
On my point of view, i think we should have this public so that the guest and the buyer can all see related products. Apart from that, the functionality is working well.

Reply:
Thanks for the comment, let me fix it